--- tripleo-heat-templates/extraconfig/tasks/yum_update.sh.orig 2017-03-21 10:35:28.301171836 -0400
+++ tripleo-heat-templates/extraconfig/tasks/yum_update.sh      2017-03-21 07:23:52.785171836 -0400
@@ -80,6 +80,8 @@
     yum deplist openstack-puppet-modules | awk '/dependency/{print $2}' | xargs yum -q -y update
     echo "Upgrading other packages is handled by config management tooling"
     echo -n "true" > $heat_outputs_path.update_managed_packages
+    yum install -y contrail-tripleo-puppet puppet-contrail
+    rsync -a /usr/share/contrail-tripleo-puppet/ /usr/share/openstack-puppet/modules/tripleo/
     exit 0
 fi
